### PR TITLE
Adds policy for ingestor role

### PR DESCRIPTION
**Pull request by kenoir**
That allows access to SQS
In order that it can poll the ingestion queue and process updates to
the API

### CODE REVIEW

File: terraform/iam.tf [hunk 1]
```diff
@@ -30,3 +30,19 @@ resource "aws_iam_role_policy" "ecs_jenkins_task" {
 }
 EOF
 }
+
+resource "aws_iam_role_policy" "ecs_ingestor_task" {
+  name = "tf_ecs_task_ingestor_policy"
```
Comment (jamesgorrie) [inline terraform/iam.tf]: What's `tf_` for? Terraform?

File: terraform/iam.tf [hunk 2]
```diff
@@ -30,3 +30,19 @@ resource "aws_iam_role_policy" "ecs_jenkins_task" {
...
+  role = "${module.ingestor.task_role_name}"
+
+  policy = <<EOF
+{
```
Comment (jamesgorrie) [inline terraform/iam.tf]: I prefer these, up to you:
https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html

File: terraform/iam.tf (hunk 1 above)
Comment (kenoir) [inline terraform/iam.tf]: Ja. Kind of pointless. Will rename.

File: terraform/iam.tf (hunk 2 above)
Comment (kenoir) [inline terraform/iam.tf]: Yup.

**Pull request merged**